terragrunt/aws/api/kms.tf

```diff
@@ -1,8 +1,5 @@
 
 data "aws_iam_policy_document" "kms_policies" {
-  # checkov:skip=CKV_AWS_109: `resources=["*"]` references the key the policy is attached to
-  # checkov:skip=CKV_AWS_111: `resources=["*"]` references the key the policy is attached to
```

Comment:
nit: could you go back to having these suppression comments here instead of in the `checkov.yml`.  Reason being that normally we do want to catch `resources=["*"]` warnings, but in this case it's a false positive.

Reply:
makes sense! Moved back